### ISSUE TITLE
Updating to ARCore 1.2

### ISSUE THREAD

**Issue by ufktvfk** [Unity-Technologies/experimental-ARInterface#73]
Hi, Does anyone know how try update this library to ARCore 1.2?

**Comment by ufktvfk** [comment]
Thank you.  Is ARInterface updated for ARKit also?

**Comment by iBicha** [comment]
No, ARKit 1.5 features (such as image markers) are not implemented in the interface.

**Comment by oliverellmers** [comment]
Wondering if anybody has given @ufktvfk 's idea a go yet? would love to be able to use the latest arkit and core together?

Cheers :)
